spring-xd-runner/pom.xml

```diff
@@ -8,7 +8,7 @@
 	<version>1.0.0.BUILD-SNAPSHOT</version>
 	<packaging>jar</packaging>
 
-	<name>spring-xd-runner</name>
+	<name>spring-xd-module-runner</name>
```

Comment:
sorry, I had to do a manual merge due to the repo and name changes. I should revert.